core/builtin.rbs

```diff
@@ -170,6 +170,10 @@ interface _Inspect
   def inspect: () -> String
 end
 
+interface _IsA
```

Comment:
Needed for `Pathname#relative_path_from`, and I think would be common enough that putting it in `builtin.rbs` is warranted. I could be persuaded though and put it in `Pathname`